packages/web/src/questionnaire/trackStructureWorkforce.ts

```diff
@@ -1,159 +1,163 @@
-const nextExceptions = [
+import type { NextTrackRules, Track } from '@/types'
+import { ConditionOperators, DataMappingFrom, HasInputOptions, TrackComponents, TrackId } from '@/types'
+import { Entreprise } from '@/types/publicodesObjects'
+
+const nextExceptions: NextTrackRules[] = [
   {
     help: "Goes to track_sectors if : doesn't have infos about sector",
     rules: [
       {
-        from: 'usedTracks',
+        from: DataMappingFrom.UsedTracks,
         id: 'siret',
         dataField: 'siret',
         conditions: [
           {
             type: 'siret',
-            operator: 'inexists'
+            operator: ConditionOperators.inexist,
```

Comment:
Was previously the case, but `inexist` does not seem to be a commonly used verb (see [dictionary entry](https://www.merriam-webster.com/dictionary/inexist) : "archaic"), and with a different meaning than intended. 

Proposed new name `isMissing` or `missing`

Reply:
Oui, je suis d'accord avec toi ! Cependant, je souhaitais limiter les changements de codes qui n’étaient pas de lors du typage ! La PR est déjà assez important ;-)

Reply:
issue: ok pour en faire une issue séparée, pour les mêmes raisons que @yvalentin. `missing` est sans doute un meilleur nommage qui correspond mieux à ce que j'avais en tête en listant cette condition.

Reply:
Done here: https://github.com/betagouv/transition-ecologique-entreprises-widget/issues/238